code/game/objects/items/binoculars.dm

```diff
@@ -52,6 +54,11 @@
 			to_chat(usr, "<span class='notice'>They are currently set to railgun targeting mode.</span>")
 		if(MODE_ORBITAL)
 			to_chat(usr, "<span class='notice'>They are currently set to orbital bombardment mode.</span>")
+	to_chat(usr, "<span class='notice'>Use on a mortar to link it for remote targeting.</span>")
+	if(linked_mortar)
+		to_chat(usr, "<span class='notice'>They are currently linked to a mortar.</span>")
+		return
+	to_chat(usr, "<span class='notice'>They are not linked to a mortar.</span>")
```

Comment:
This should use the passed user arg, I will also appreciate if you change the other ones